tests/framework/db/mysql/QueryBuilderTest.php

```
@@ -807,38 +800,32 @@ public function testDropPrimaryKeyWithQualifiedTableNames(
         );
     }
 
-    public function testRenameColumnRetainsInlineCheckInGeneratedDefinition(): void
+    public function testRenameColumnBuildsNativeSqlForColumnWithInlineCheck(): void
     {
         $db = $this->getConnection(false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)